adminSiteServer/adminRouter.tsx

```diff
@@ -130,7 +131,14 @@ adminRouter.get("/datasets/:datasetId.csv", async (req, res) => {
     ).name
     res.attachment(filenamify(datasetName) + ".csv")
 
-    return Dataset.writeCSV(datasetId, res)
+    const writeStream = new Writable({
```

Comment:
Perhaps there's a better way how to work with streams that emit chunks (and need calling `.toString()`)? I've tried a few things and nothing worked (could be polars issue).